src/test/resources/dmtf/envelope_upgraded.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ovf:Envelope xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:vcloud="http://www.vmware.com/vcloud/v1.5" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vmw="http://www.vmware.com/schema/ovf" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2.22.0/CIM_VirtualSystemSettingData.xsd http://www.vmware.com/schema/ovf http://www.vmware.com/schema/ovf http://schemas.dmtf.org/ovf/envelope/1 http://schemas.dmtf.org/ovf/envelope/1/dsp8023_1.1.0.xsd http://www.vmware.com/vcloud/v1.5 http://emea01.canopy-cloud.com/api/v1.5/schema/master.xsd http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2.22.0/CIM_ResourceAllocationSettingData.xsd">
+<ovf:References/>
+<ovf:NetworkSection>
+    <ovf:Info>The list of logical networks</ovf:Info>
+    <ovf:Network ovf:name="vAppNet-Deployment_Network_001">
+        <ovf:Description/>
+    </ovf:Network>
+</ovf:NetworkSection>
+<vcloud:CustomizationSection goldMaster="false" ovf:required="false">
+    <ovf:Info>VApp template customization section</ovf:Info>
+    <vcloud:CustomizeOnInstantiate>true</vcloud:CustomizeOnInstantiate>
+</vcloud:CustomizationSection>
+<vcloud:NetworkConfigSection ovf:required="false">
+    <ovf:Info>The configuration parameters for logical networks</ovf:Info>
+    <vcloud:NetworkConfig networkName="vAppNet-Deployment_Network_001">
+        <vcloud:Description/>
+        <vcloud:Configuration>
+            <vcloud:IpScope>
```

Comment:
seems that it used to be `IpScopes` but now it is missing. Which vCD version is producing this one?